modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc

```diff
@@ -103,14 +105,115 @@ Like the Cypher deprecations table, the last column links to a view of this spec
+
+image::mrr-finalize-test.png[width=500]
+
+To save costs remember to delete the test instance after manual testing is done.
+You can also choose to skip this step and keep the test instance as long as you want.
+
+=== Run the live migration
+
+Use the "Live migration" buttons at the top or bottom of the page and then follow the steps outlined in the dialog boxes.
+
+The steps of running the live migration are:
+
+* Carefully read and act upon the steps described in the "Read before live migration" dialog.
+Proceed only if you made the appropriate preparations (e.g. backups of your configurations).
+* Carefully read and act upon the step described in the "Writes made on the v4 instance during migration" dialog.
+Make sure that your application will not write to the original instance during the migration to prevent this data from being lost.
+* Configure a target instance, as described in the next section.
+** If you have selected a new instance to migrate to: Download the new credentials for that instance.
+* Wait for the migration to finish.
+* Follow all steps outlined in "Next steps before finalizing the live migration" at the top of the Migration Readiness Report page.
+This includes all your testing on the migrated instance.
+* Once you are done with testing, click the "Finalize live migration" button and complete the dialog to remove your original version 4 instance.
+
+There can only be one live migration in progress at any time.
+If you need to, you can restart the process at any point by removing the migrated instance until you finalize the migration by removing the original instance.
+
+==== Configure target instance
+
+An instance can either be migrated to a brand-new instance or an instance that is already running the latest version of Aura and that fits the memory and storage configuration of the original instance.
+This means that if you choose the second option the instance you want to migrate to has to have at least the same amount of memory and storage as the original one.
+
+Note that cloning into an existing instance will overwrite all of its existing data and name.
+This action cannot be undone and may take longer than cloning to a new instance.
+It is advised to take a snapshot and download it before continuing if you still have data on that instance you want to keep.
+
+Regardless of which option you choose, the name of the migration target instance will be the same as the original instance.
+
+==== Testing the migrated instance
+
+Once you see the following box on the Migration Readiness Report your migrated instance is ready for testing.
+Follow the steps described and test your instance to make sure your application can work with it in your production system.
+
+image::mrr-live-migration-ready-for-test.png[]
+
+==== Finalize live migration
+
+Once you are done with testing, click the "Finalize live migration" button and the following dialog will be displayed.
```

Comment:
```suggestion
Once you are done with testing, use the "Finalize live migration" button.
You will be asked to acknowledge the finalization since the original instance is permanently removed in the process.
Additionally, when the dialog is completed, you will no longer have access to the Migration readiness report.
```